gp-includes/routes/translation.php

```diff
@@ -28,7 +28,11 @@ public function import_translations_get( $project_path, $locale_slug, $translati
 			return $this->die_with_404();
 		}
 
-		if ( $this->cannot_and_redirect( 'approve', 'translation-set', $translation_set->id ) ) {
+		$can_import = $this->can( 'import', 'translation-set', $translation_set->id );
+		$can_only_import_waiting = ! $can_import && $this->can( 'import-waiting', 'translation-set', $translation_set->id );
+
+		if ( ! $can_import && ! $can_only_import_waiting ) {
```

Comment:
Done in https://github.com/GlotPress/GlotPress-WP/pull/378/files#diff-d32355c0a0d15e6d80d430ba54ea3362R211 and https://github.com/GlotPress/GlotPress-WP/pull/378/files#diff-c4239e7606696e35a13d1d0811859f71R237